app/pages/Import/TaskCreate/PasswordInputModal/index.tsx

```diff
@@ -33,7 +33,7 @@ const PasswordInputModal = (props: IProps) => {
         onChange={e => setPassword(e.target.value)}
       />
       <div className={styles.btns}>
-        <Button onClick={() => handleConfirm()}>
```

Comment:
use handleCancel, it will not clear password if use onCancel

Reply:
this is props event